components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -317,6 +319,20 @@ export function CreateWorkspacePage() {
         );
     }
 
+    if (needsGitAuthorization) {
+        return (
+            <div className="flex flex-col mt-32 mx-auto ">
```

Comment:
It might be nice to use the `<EmptyMessage/>` component here (`components/dashboard/src/components/EmptyMessage.tsx`).

Reply:
I'd need to change its API to make it fit (e.g. multiple buttons). @gtsiolis the diesgn of that is different to what you proposed here. Is that intentional?

Reply:
> @gtsiolis the design of that is different to what you proposed here. Is that intentional? Maybe we want to align this in a follow-up PR?

@svenefftinge Yes, made some facelift changes but should be fine as is. I can make the style changes in a separate PR. Reusing the Empty State pattern could also be nice.